src/backend/base/langflow/api/v1/schemas.py

```diff
@@ -287,7 +287,10 @@ def _serialize_and_truncate(obj: Any, max_length: int = MAX_TEXT_LENGTH) -> Any:
                 return f"{obj[:max_length]}... [truncated]"
             return obj
         if isinstance(obj, datetime):
-            return obj.astimezone().isoformat()
+            # Ensure the datetime object is timezone-aware before calling astimezone()
+            if obj.tzinfo is None:
+                obj = obj.replace(tzinfo=timezone.utc)
+            return obj.astimezone(timezone.utc).isoformat()
```

Comment:
Calling `astimezone` will change the time.

Just replace the timezone.
```suggestion
            return obj.replace(tzinfo=timezone.utc).isoformat()
```